amps/ags/rm-automation/rm-automation-community-rest-api/src/test/java/org/alfresco/rest/rm/community/hold/AddToHoldsV1Tests.java

```diff
@@ -65,71 +58,68 @@ public class AddToHoldsV1Tests extends BaseRMRestTest
+        testSite = dataSite.usingAdmin().createPublicRandomSite();
         documentHeld = dataContent.usingAdmin().usingSite(testSite)
             .createContent(CMISUtil.DocumentType.TEXT_PLAIN);
-        dataContent.usingAdmin().usingSite(testSite)
-            .createContent(CMISUtil.DocumentType.TEXT_PLAIN);
-        dataContent.usingAdmin().usingSite(testSite)
-            .createContent(CMISUtil.DocumentType.TEXT_PLAIN);
 
         STEP("Add the content to the hold.");
         getRestAPIFactory()
             .getHoldsAPI(getAdminUser())
             .addChildToHold(HoldChild.builder().id(documentHeld.getNodeRefWithoutVersion()).build(), hold.getId());
-
-        STEP("Create users");
-        UserModel userAddHoldPermission = roleService.createUserWithSiteRoleRMRoleAndPermission(testSite,
-            UserRole.SiteCollaborator, holdNodeRef, UserRoles.ROLE_RM_MANAGER, PERMISSION_FILING);
-        users.add(userAddHoldPermission);
     }
 
     @Test
     public void retrieveTheContentOfTheHoldUsingV1API() throws Exception
     {
         STEP("Retrieve the list of children from the hold and collect the entries that have the name of the active " +
             "content held");
-        List<RestNodeModel> documentsHeld = restClient.authenticateUser(getAdminUser()).withCoreAPI()
+        List<String> documentNames = restClient.authenticateUser(getAdminUser()).withCoreAPI()
             .usingNode(toContentModel(holdNodeRef))
             .listChildren().getEntries().stream()
-            .filter(child -> child.onModel().getName().contains(documentHeld
-                .getName()))
-            .collect(Collectors.toList());
+            .map(RestNodeModel::onModel)
+            .map(RestNodeModel::getName)
+            .toList();
+
         STEP("Check the list of active content");
-        assertEquals(documentsHeld.size(), 1, "The active content is not retrieve when getting the children from the " +
+        assertEquals(documentNames, Set.of(documentHeld.getName()));
+        assertEquals(documentNames.size(), 1, "The active content is not retrieve when getting the children from the " +
             "hold folder");
-        assertEquals(documentsHeld.get(0).onModel().getName(), documentHeld.getName());
     }
 
     @Test
     public void retrieveTheHoldWhereTheContentIsAdded()
     {
         RestNodeAssociationModelCollection holdsEntries = getRestAPIFactory()
             .getNodeAPI(documentHeld).usingParams("where=(assocType='rma:frozenContent')").getParents();
-        Hold hold = getRestAPIFactory().getHoldsAPI(getAdminUser())
+        Hold retrievedHold = getRestAPIFactory().getHoldsAPI(getAdminUser())
             .getHold(holdsEntries.getEntries().get(0).getModel().getId());
-        assertTrue(hold.getName().contains(HOLD), "Could not find " + "hold with name " + HOLD);
+        assertEquals(retrievedHold, hold,
+            "Holds are not equal. Expected:<" + hold.toString() + "> but was:<" + retrievedHold.toString());
```

Comment:
I think that `assertEquals` should already give you the string representation of these two holds.

Reply:
Right. The string representation of these two holds has been removed.